_life/money.md

```diff
@@ -79,75 +71,74 @@ Do not expect good yields from non-investment bank accounts, even the savings on
+> ⚠️ The remainder of this page is a helpful guide to common taxable items and deductions _in the canton of Vaud_, not tax advice about your personal situation.
+> You should still read every field in the tax form at least once to ensure you do not forget anything.
 
-If you have a B permit but do not meet one of the criteria above, typically because you have a 3a pillar, you must request an "ulterior ordinary taxation", or "taxation ordinaire ultérieure" in French.
+If you have a B permit but do not meet one of the criteria explained at the beginning of this page, typically because you have a 3a pillar, you must request an "ulterior ordinary taxation", or TOU "taxation ordinaire ultérieure" in French.
 The form to request one is on the [Vaud website](https://www.vd.ch/themes/etat-droit-finances/impots/pour-les-employeurs/impot-a-la-source/personnes-imposees-a-la-source-sourciers/).
-Once this is done, you keep being taxed at the source but you follow the procedure detailed next to declare your taxes. This cannot be undone, once requested you will declare your taxes every year.
+Once this is done, you keep being taxed at the source but you follow the procedure detailed next to declare your taxes.
+This cannot be undone, once requested you will declare your taxes every year.
 
-If you have to file your taxes, typically because you requested so (above) or because you are Swiss, you will receive a letter at the start of the next year with information.
-You then need to file your taxes using [VaudTax](https://www.vd.ch/themes/etat-droit-finances/impots/impots-pour-les-individus/remplir-ma-declaration-dimpot/prestation-vaudtax/), a website which is basically a giant form.
+If you have to file your taxes because you requested so (above) or because you are a permanent resident, you will receive a letter when tax season comes.
+This letter contains identifiers for [VaudTax](https://www.vd.ch/themes/etat-droit-finances/impots/impots-pour-les-individus/remplir-ma-declaration-dimpot/prestation-vaudtax/), a website which is basically a giant form.
+**VaudTax offers a "simulation" mode** that does not need identifiers, with which you can use to decide whether requesting a TOU is worth it or not if you are not legally required to do so but wonder whether you could benefit from extra deductions.
+Remember that switching to a TOU is permanent, so your situation in terms of deductions should be stable long-term.
 
-You need to declare your information, income, and wealth. To do so, first get the following documents and information:
-- The letter you got from the tax office that includes your "n° de contribuable" and the "code de contrôle" (the latter changes each year)
+Before opening VaudTax, get the following documents and information, so that filing your taxes becomes a quick 15 minutes affair:
+- The letter you got from the tax office that includes your "n° de contribuable" and "code de contrôle" identifiers (the latter changes each year)
```

Comment:
Oops! Looks like you're using the degree symbol (U+00B0 "Degree Sign") to write the abbreviated form of the word *numéro*. Instead, the typographically correct symbol to use is the superscript lowercase o (U+1D52 "Modifier Letter Small O"), as follows: nᵒ. ([source](https://french.stackexchange.com/questions/41992/which-glyphs-should-be-used-to-represent-the-word-numero-n-n-or#41996), for example)

Reply:
Damn, good thing you spotted this before the Unicode police caught me! It's too bad Swiss keyboards have the former and not the latter.